mantidproject/mantid
# Add and improve documentation for absorption corrections

**To test:**

Review the documentation changes to ensure they make sense.

I had to make a small change to `SetSample` so that the `Axis` argument was interpreted as an integer and not a double. I only picked this up when writing the usage tests.

Fixes #16044 



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
